agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/DynamicConfiguration.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.agent.internal.init;
+
+import com.microsoft.applicationinsights.agent.internal.configuration.Configuration;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DynamicConfiguration {
```

Comment:
I would prefer something more explicit and from a user point of view, `ProgrammaticConfiguration`?

Reply:
`RuntimeConfiguration`?